runtime/src/runtime/mod.rs

```diff
@@ -245,6 +245,9 @@ pub trait Runtime: Primitives + Verifier + RuntimePolicy {
 
     /// Returns the gas base fee (cost per unit) for the current epoch.
     fn base_fee(&self) -> TokenAmount;
+
+    /// Returns true if the system is in read-only mode.
```

Comment:
I'd say "if the call is read-only". And can we briefly describe what that means here? Will writes be rejected, or merely ignored after returning?